modules/iceoryx/2.95.3/MODULE.bazel

```diff
@@ -0,0 +1,15 @@
+module(
+    name = "iceoryx",
+    version = "2.95.3",
+    bazel_compatibility = [">=7.2.1"],
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "cpptoml", version = "0.1.1")
+bazel_dep(name = "platforms", version = "0.0.10")
+bazel_dep(name = "rules_cc", version = "0.0.9")
+bazel_dep(
+    name = "buildifier_prebuilt",
```

Comment:
Can this be a dev dep? Same for googletest.

Reply:
No: https://github.com/eclipse-iceoryx/iceoryx/pull/2371